prisma/seedDev.mjs

```diff
@@ -39,6 +39,17 @@ function randomAddress(districts, isPrimary) {
   };
 }
 
+function generateSocialMediaLinks() {
+  const socialMediaList = ['facebook', 'instagram', 'youtube', 'linkedin', 'tiktok', 'viber', 'telegram'];
+  const subsetLength = Math.floor(Math.random() * 5) + 1;
+  const shuffledList = socialMediaList.sort(() => Math.random() - 0.5);
+
+  return shuffledList.slice(0, subsetLength).reduce((obj, cur) => ({
+    ...obj,
+    [cur]: faker.internet.url(),
+  }), {});
```

Comment:
You could do this in a more "functional"-way:

```javascript
function generateSocialMediaLinks() {
  return ['facebook', 'instagram', 'youtube', 'linkedin', 'tiktok', 'viber', 'telegram']
    .sort(() => Math.random() - 0.5)
    .slice(0, Math.floor(Math.random() * 5) + 1)
    .reduce((obj, cur) => ({ ...obj, [cur]: faker.internet.url() }), {});
}
```

Or even you can simplify code to omit `reduce` (that's optional and more a matter of preference):

```javascript
function generateSocialMediaLinks() {
  return Object.fromEntries(
    ['facebook', 'instagram', 'youtube', 'linkedin', 'tiktok', 'viber', 'telegram']
      .sort(() => Math.random() - 0.5)
      .slice(0, Math.floor(Math.random() * 5) + 1)
      .map(network => [network, faker.internet.url() ])
  );
}
```

Reply:
Thanks! Will take this into account next time.

Decided to go this way:
```js
function generateSocialMediaLinks() {
  const socialMediaList = ['facebook', 'instagram', 'youtube', 'linkedin', 'tiktok', 'viber', 'telegram'];

  return Object.fromEntries(socialMediaList
    .sort(() => Math.random() - 0.5)
    .slice(0, Math.floor(Math.random() * 5) + 1)
    .map(network => [network, faker.internet.url()])
  );
}
```